pyspline/pyVolume.py

```diff
@@ -830,6 +830,11 @@ def projectPoint(self, x0, nIter=25, eps=1e-10, **kwargs):
             Maximum number of Newton iterations to perform
         eps : float
             Tolerance for the Newton iteration
+        volBounds : list of lists
+            Optional input to prescribe bounds to the parametric coordinates during the projection.
+            The value is a list, that should contain 3 lists with 2 float entries each.
```

Comment:
No comma, replace "should" with "must" as this does not work with anything less than the full list, right?

Reply:
Looks like you committed the wrong file :eyes: